pkg/data/account.go

```diff
@@ -0,0 +1,172 @@
+package data
+
+import (
+	gErrors "errors"
+	"fmt"
+	"github.com/Optum/dce/pkg/errors"
+	"github.com/Optum/dce/pkg/model"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+)
+
+// Account - Data Layer Struct
+type Account struct {
+	AwsDynamoDB    dynamodbiface.DynamoDBAPI
```

Comment:
Can this just be called `DynamoDB` (are there other providers of DynamoDB than AWS?)